testing/test_column_line_numbers.py

```diff
@@ -0,0 +1,57 @@
+import ast
+from pathlib import Path
+from typing import List, Tuple
+
+import pytest
+from flake8_annotations import checker
+
+
+TEST_FILE = Path("./testing/code/column_line_numbers.py")
+
+# (line, column) tuples where we should get linting errors in the test file
+# Line numbers are 1-indexed
+# Column offsets are 0-indexed when yielded by our checker; flake8 adds 1 when emitted
+SHOULD_ERROR = ((16, 8), (16, 11), (24, 4), (25, 4), (26, 2))
```

Comment:
This is a fragile way of doing things but I can't think of other ways.

Reply:
This mirrors what other plugins do (e.g. [bugbear](https://github.com/PyCQA/flake8-bugbear/tree/master/tests)) but I don't disagree that it's fragile. It's probably worth adding a note to the testing source that these values need to be adjusted if the source changes. This might be another good spot for a namedtuple too.